### DISCUSSION TITLE
Bot tries to create its congig in system32

### DISCUSSION THREAD

**Discussion by ghost** [jagrosh/MusicBot#912] [Troubleshooting] [answered]
What could be the cause of this? The bot.folder is located at Documents, and i have redownloaded the config and the bot multiple times, but to no avail

**Comment by MichailiK** [comment, marked as answer]
Don't use right click -> open with to open the MusicBot. For some reason, Windows sets the current directory to system32 when you do that. Do right click -> open with -> select another app -> find java in the list -> tick the checkbox at the bottom

**Reply by ghost** [reply]
thank you!
